PowerEditor/visual.net/notepadPlus.vcxproj

```diff
@@ -530,6 +530,11 @@ copy ..\src\contextMenu.xml ..\binarm64\contextMenu.xml
   <ItemGroup>
     <Image Include="..\src\icons\allChars_off.ico" />
     <Image Include="..\src\icons\allChars_on.ico" />
+    <Image Include="..\src\icons\asciiPanel.ico" />
+    <Image Include="..\src\icons\asciiPanel2.ico" />
+    <Image Include="..\src\icons\chameleon.ico" />
+    <Image Include="..\src\icons\clipboardPanel.ico" />
+    <Image Include="..\src\icons\clipboardPanel2.ico" />

```

Comment:
All the diff in `notepadPlus.vcxproj` should be restored.
There's no reason to modify project file.